libnavigation-base/src/main/java/com/mapbox/navigation/base/internal/route/NavigationRouteEx.kt

```diff
@@ -147,7 +183,16 @@ fun NavigationRoute.update(
         .routes(refreshedRoutes)
         .directionsResponseBlock()
         .build()
-    return copy(directionsResponse = refreshedResponse)
+    return copy(directionsResponse = refreshedResponse, expirationTime = newExpirationTime)
+}
+
+fun NavigationRoute.updateExpirationTime(newExpirationTime: Long?): NavigationRoute {
```

Comment:
```suggestion
fun NavigationRoute.updateExpirationTime(expirationTimeEllapsedSystemsSeconds: Long?): NavigationRoute {
```

Or maybe introduce a type for that:
```
@JvmInline
value class ElapsedSystemSeconds(val value: Long)
```
?

Because right now it's clear that this is time, but it's not clear how to use it.